src/test/cypress/support/pageobjects/ArtemisPageobjects.ts

```diff
@@ -26,14 +31,23 @@ export class ArtemisPageobjects {
     navigationBar = new NavigationBar();
     examCreation = new ExamCreationPage();
     examManagement = new ExamManagementPage();
+    examStartEnd = new ExamStartEndPage();
+    examNavigationBar = new ExamNavigationBar();
+    programmingExerciseCreation = new ProgrammingExerciseCreationPage();
```

Comment:
Why was this added? Isn't this a duplicate of `programmingExercise.creation`?

Reply:
Good catch. Yeah its a duplicate. I removed it